database/db.go

```diff
@@ -231,10 +231,10 @@ func (db *database) CreateSchema(ctx context.Context) (err error) {
         );
  		CREATE TABLE IF NOT EXISTS season_rewards(
             id          SERIAL PRIMARY KEY       NOT NULL,
-            season_id   BYTEA                    NOT NULL,
```

Comment:
we need season_id in table season_rewards for history that Ruslan was talking about